system-test/spanner.ts

```diff
@@ -124,15 +129,42 @@ describe('Spanner', () => {
         `Not creating temp instance, using + ${instance.formattedName_}...`
       );
     }
-    const [, googleSqlOperation1] = await DATABASE.create({
-      schema: `
+
+    if (IS_EMULATOR_ENABLED) {
+      const [, googleSqlOperation1] = await DATABASE.create({
+        schema: `
           CREATE TABLE ${TABLE_NAME} (
             SingerId STRING(1024) NOT NULL,
             Name STRING(1024),
           ) PRIMARY KEY(SingerId)`,
-      gaxOptions: GAX_OPTIONS,
-    });
-    await googleSqlOperation1.promise();
+        gaxOptions: GAX_OPTIONS,
+      });
+      await googleSqlOperation1.promise();
+    } else {
+      // Reading proto descriptor file
+      const protoDescriptor = fs
+        .readFileSync('test/data/descriptors.pb')
+        .toString('base64');
+
+      const [, googleSqlOperation1] = await DATABASE.create({
+        schema: [
+          `
+        CREATE PROTO BUNDLE (
```

Comment:
Why this Proto is not added for Emulator. Is it not supported in emulator ?

Reply:
There are issues in emulator which is not yet addressed. Will unskip the tests in a seperate PR when emulator is ready.